src/sonic-bgpcfgd/bgpcfgd/main.py

```diff
@@ -59,6 +60,8 @@ def do_work():
         BBRMgr(common_objs, "CONFIG_DB", "BGP_BBR"),
         # Static Route Managers
         StaticRouteMgr(common_objs, "CONFIG_DB", "STATIC_ROUTE"),
+        # Route Advertisement Managers
+        AdvertiseRouteMgr(common_objs, "STATE_DB", "ADVERTISE_NETWORK"),
```

Comment:
Is this defined in the schema? if not, could you please raise a PR?

Reply:
Defined the table in the schema. Pending on swss-common submodule upgrade https://github.com/Azure/sonic-buildimage/pull/9275. PR test would fail before that one merges.